bip-0324.mediawiki

```diff
@@ -473,24 +475,24 @@ class FSChaCha20:
 
 Encryption and decryption of packets then follow by composing the ciphers from the previous section as building blocks.
 
-<pre>
+<source lang="python">
 LENGTH_FIELD_LEN = 3
 HEADER_LEN = 1
 IGNORE_BIT_POS = 7
 
-def v2_enc_packet(peer, contents, aad=b'', ignore=False):
+def v2_enc_packet(peer, contents, aad=b"", ignore=False):
     assert len(contents) <= 2**24 - 1
     header = (ignore << IGNORE_BIT_POS).to_bytes(HEADER_LEN, 'little')
     plaintext = header + contents
     aead_ciphertext = peer.send_P.encrypt(aad, plaintext)
-    enc_contents_len = peer.send_L.encrypt(len(contents).to_bytes(LENGTH_FIELD_LEN, 'little'))
+    enc_contents_len = peer.send_L.crypt(len(contents).to_bytes(LENGTH_FIELD_LEN, 'little'))
     return enc_contents_len + aead_ciphertext
-</pre>
+</source>
 
-<pre>
+<source lang="python">
 CHACHA20POLY1305_EXPANSION = 16
 
-def v2_receive_packet(peer, aad=b'', skip_decoy=True):
```

Comment:
nit: Did you change these intentionally? Now we have both `'` and `"` in the source, which is also not very consistent.